examples/cloudbuild-selective-deployment/cloudbuild.yaml

```diff
@@ -0,0 +1,53 @@
+timeout: 1200s
+substitutions:
+  _TF_SA_EMAIL: ''
+  _PREVIOUS_COMMIT_SHA: ''
```

Comment:
Nope, commit will be calculated in delta-changes.sh. But one can provide a manually specified commit here or in cloud build substitution variables from the console and it would run based on that instead.